### DISCUSSION TITLE
How do I get the font width metadata property using Fonttools?

### DISCUSSION THREAD

**Discussion by futuremotiondev** [fonttools/fonttools#3142] [General]
I'm looking for a way to retrieve this property for a script I'm writing:

![FontExpert_VjazWR7WJy](https://github.com/fonttools/fonttools/assets/33441569/331d60aa-9b8f-487c-bdd2-3bbd9ae75fd9)

What is the correct way to accomplish this using FontTools?

**Comment by ftCLI** [comment]
This is not an issue.

Anyway:

```python
font = TTFont(file)
width = font['OS/2'].usWidthClass
```
`width` is an integer (1-9), and you can map them as you prefer. For example: https://learn.microsoft.com/en-us/typography/opentype/spec/os2#uswidthclass

**Reply by futuremotiondev** [reply]
Will this work for both TTF and OTF fonts? Sorry for such a simple question, just trying to get a grasp on this.

**Reply by ftCLI** [reply]
Yes, both have OS/2 table
